src/helpers/youtubeFetcher.js

```diff
@@ -0,0 +1,30 @@
+const fetch = require('node-fetch');
+const { YOUTUBE_API_KEY, YOUTUBE_CHANNEL_ID } = require('../config');
+
+function transformYoutubeResponseData(data) {
+  return data.items.map((item) => {
+    const { title, publishedAt, description } = item.snippet;
+    return {
+      type: 'video',
+      name: title,
+      videoID: item.id.videoId,
+      date: publishedAt,
+      url: `https://www.youtube.com/watch?v=${item.id.videoId}`,
+      thumbnail: item.snippet.thumbnails.high,
+      title,
+      description,
+    };
+  });
+}
+
+export default async function fetchLatestYoutubeVideos() {
+  const url = `https://www.googleapis.com/youtube/v3/search?key=${YOUTUBE_API_KEY}&channelId=${YOUTUBE_CHANNEL_ID}&part=snippet&order=date&maxResults=50`;
+  try {
+    const resp = await fetch(url);
+    return transformYoutubeResponseData(resp.json());
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(e);
```

Comment:
We should handle this error a little differently.

Reply:
That's out of the scope of this pull request

Reply:
If you feel that it should be handled elsewhere you can create a Trello card for it. Thanks!

Reply:
Is there a common way we should log JSON errors or something? `seed.js` and `localDevSeed.js` both use `catch(error) { console.log(error) }`. I thought about telling him he should change it to `console.error`, but I see `next(e)` all over the place... not sure what that is all about... but could you be more specific with change requests

Reply:
Yeah, this was an initial review. Ill explain our error handling structure in my review tomorrow. The main difference is that the error that we are logging are for local dev and the errors passed along to the error handling middleware are for sending as a response to the client.

Reply:
I am looking through this again. It seems that this is going to be a dev only function so we don't have to worry about the error handling. We are good with how it is now.